src/components/atoms/loading/loading.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Loading = () => <p>loading...</p>;
```

Comment:
Overly simple loading component. Really we want to do skeletons of the result elements for the purposes of this project. Definitely need to find a better solution here.